OasisApi.ts

```diff
@@ -360,11 +367,13 @@ function convertHtlc<TStatus extends HtlcStatus>(htlc: RawHtlc<TStatus>): Htlc<T
 function coinsToUnits(asset: OasisAsset, value: string | number, roundUp = false): number {
     let decimals: number;
     switch (asset) {
-        case OasisAsset.EUR: decimals = 2; break;
+        case OasisAsset.EUR:
+        case OasisAsset.CRC:
+            decimals = 2; break;
```

Comment:
Are we going to handle CRC with 2 decimals? AFAIK decimals in CRC are no longer used. Does Fastspot or OASIS ever use decimals for CRC?

Reply:
We won't use decimals, but in the UI we might add two decimals with 2 zeros like `5700.00`

Reply:
Just know that this means that the library will return 500 CRC as `50000`, as it converts what the API gives to the smallest unit, and you are specifying here that the smallest unit is 2 decimals lower than the decimal dot.